aten/src/ATen/native/sparse/SparseCsrTensorMath.cpp

```diff
@@ -71,8 +71,8 @@ void convert_indices_from_coo_to_csr_cpu(const Tensor& result, const Tensor& inp
     data_out[i] = static_cast<output_t>(numel);
 }
 
-template <typename F>
-Tensor& unary_op_out(F op_out, const Tensor& self, Tensor& result) {
+template <typename F, typename ...Args>
+Tensor& unary_op_out(F op_out, const Tensor& self, Tensor& result, Args... args) {
```

Comment:
Note that you are copying both `F` (might be a reasonable thing to do if it's a poiter) and `Args` (it's not clear whether you want to do this.
You probably want to take `Args`  as a [forwarding reference](https://en.cppreference.com/w/cpp/language/reference), i.e. `Args&&...` and then `std::forward(args)...` afterwards. Same below.

Reply:
I have fixed both, @Lezcano - thank you for the review and suggestion. Hope it looks good now. :)